packages/server/graphql/private/typeDefs/Mutation.graphql

```diff
@@ -14,10 +14,54 @@ type Mutation {
     description: String!
 
     """
-    The expiration date of the feature flag (optional)
+    The expiration date of the feature flag
     """
     expiresAt: DateTime!
   ): AddFeatureFlagPayload!
+
+  """
+  Apply a feature flag to specified subjects (users, teams, or organizations)
+  """
+  applyFeatureFlag(
+    """
+    The name of the feature flag to apply
+    """
+    flagName: String!
+    """
+    The subjects to which the feature flag will be applied
+    """
+    subjects: SubjectsInput!
+  ): ApplyFeatureFlagPayload!
+
+  """
+  Delete an existing feature flag
+  """
+  deleteFeatureFlag(
```

Comment:
-1 I'm not sure what this mutation is supposed to do. Does it remove the feature flag from all subjects or does it delete the entry from the feature flag table? What happens to the static types in this case?
I think we probably don't need static types for feature flags but rather a field on Org, Team and User: `hasFeatureFlag(flagName: String!): Boolean!`. But make sure first that this actually would work.

Reply:
It removes the flag from the `featureFlag` table. 

It's not strictly necessary as we can just update the expiry date of the flag so that it no longer applies. 

I thought it'd be helpful to include so that we can clean up flag names in the db that we're no longer using, but I can remove this if it's confusing.

Reply:
I think that's fine then. It will also remove it from all owners. Maybe update the description to match it?